README.md

```diff
@@ -1,25 +1,175 @@
-# donkeycarLPH
-Organisation / Doc du projet donkeycar des LPH
+<div align="center">
+<h1> donkeycar LPH </h1>
 
-## Documentation :
+![repo-size](https://img.shields.io/github/repo-size/mdl29/donkeycarLPH?style=flat-square)
+![contributor](https://img.shields.io/github/contributors/mdl29/donkeycarLPH?style=flat-square)
+![commitActivity](https://img.shields.io/github/commit-activity/m/mdl29/donkeycarLPH?logo=github&style=flat-square)
+![mainLanguage](https://img.shields.io/github/languages/top/mdl29/donkeycarLPH?color=yellow&style=flat-square)
 
-- [Framapad](https://annuel.framapad.org/p/LPH-donkeycar)
+### In this project we present you a solution to industrialize the production of donkeycar cars, allow to everyone to install easily donkeycar environnement. We have also added some tools like our manager to make publics demonstrations, races during events to allow public to drive, race and understand the process of machine learning
+</div>
+<br>
 
-## Software :
+ > 💡 A fork of the original [donkeycar project](https://github.com/autorope/donkeycar)
+ 
+ <br>
 
-titre | description | fichier
- ---: | :---: | :--- 
-Doc pour le boutton poussoir | Créer un boutton permettant d'éteindre une voiture | [documentation](doc/boutton/Boutton_poussoir.md)
-Doc pour les backups | Comment créer sa backup pour sa raspberry pi | [documentation](doc/backup/car-backup-procedure.md)
-Doc pour Ansible | Comment installer donkeycar sur des voitures ( plus nos features ) avec Ansible en ssh | [documentation](ansible/README.md)
-Doc pour packer | Comment generer une image propre pour les voitures avec packer | [documentation](packer/README.md)
+<details>	
+  <summary><b> 🚀 How to use our project</b></summary>
+    
+## 🧪 Create your ansible recipe
 
-## Hardware :
+*** 
 
-titre | description | fichier
- ---: | :---: | :--- 
-Shéma des branchements| Schema de la connectique de la donkeycar | [image](doc/schema/schema-electronique.png) / [Fichier Fritzing](doc/schema/shema-electronique.fzz)
-Antivol | Antivol pour manette PS4 | [doc](3dDesigns/anti-theft-ps4/README.md)
-Chassis pour voiture V4 | Picots pour adaptation de la voiture au chassis donkeycar | [doc](3dDesigns/car-chassis/README.md)
-Camera cap holder | A camera cap holder on the chassis for avoid to lose ours camera caps | [doc](3dDesigns/camera-cover/README.md)
+First you should create your own recipe book with your wanted recipes. Indeed, ansible have a configuration file where we specifie what options ( recipe ) we want, so you have to modifie the [donkeycar.yml](ansible/donkeycar.yml) file and comments recipe who you don't want
```

Comment:
The "indeed" sounds a bit weird, but you can keep it if you want.
ansible *has* ~~have~~ a configuration ... *modify* ~~modifie~~ ... comment *recipies which* ~~recipie who~~.